.github/workflows/ms.network.firewallpolicies.yml

```
@@ -128,7 +163,8 @@ jobs:
       - name: Set environment variables
         uses: deep-mm/set-variables@v1.0
         with:
-          variableFileName: 'variables.module'
+          # Name of variable file
+          variableFileName: 'variables.module' # Don't write .json here
```

Need to change ALL of these based on the rename to the var file.

```suggestion
          variableFileName: 'global.variables' # Don't write .json here
```

---

done